src/cli/Cli.ts

```diff
@@ -418,41 +415,43 @@ export class Cli {
    * Loads command from the specified file into CLI. If can't find the file
    * or the file doesn't contain a command, loads all available commands.
    *
-   * @param commandFilePath File path of the file with command to load
+   * @param commandFilePathUrl File path of the file with command to load
```

Comment:
```suggestion
   * @param commandFileUrl File path of the file with command to load
```

Reply:
The arg is actually command file path, so I'll need to revert this change